src/components/icons/BTC.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
```

Comment:
i noticed we have this import in all icons file, do we need them?

Reply:
updated to remove all imports from icons file